### ISSUE TITLE
CI is currently red and signalizes invalid markdown files:

### ISSUE THREAD

**Issue by maxhbr** [spdx/spdx-3-model#183]
see https://github.com/spdx/spdx-3-model/actions/runs/4634281022/jobs/8200331086, the errors are:

```
Run cd spec-parser
ERROR   : ../model/Software/Classes/File.md:Ln 32,Col 2: Syntax Error
	 Line no 32: '##'
ERROR   : Unable to parse `Class` markdown: '../model/Software/Classes/File.md'
ERROR   : ../model/Software/Classes/Package.md:Ln 55,Col 2: Syntax Error
	 Line no 55: '##'
ERROR   : Unable to parse `Class` markdown: '../model/Software/Classes/Package.md'
ERROR   : ../model/Build/Classes/Build.md:Ln 49,Col 2: Syntax Error: Expected `<key>: <values>`
	 Line no 49: '  - minCount:0\n'
ERROR   : ../model/Build/Classes/Build.md:Ln 50,Col 2: Syntax Error: Expected `<key>: <values>`
	 Line no 50: '  - maxCount:1\n'
ERROR   : ../model/Build/Classes/Build.md:Ln 53,Col 2: Syntax Error: Expected `<key>: <values>`
	 Line no 53: '  - minCount:0\n'
ERROR   : ../model/Build/Classes/Build.md:Ln 54,Col 2: Syntax Error: Expected `<key>: <values>`
	 Line no 54: '  - maxCount:1\n'
ERROR   : Unable to parse `Class` markdown: '../model/Build/Classes/Build.md'
ERROR   : ../model/Core/Classes/SpdxDocument.md:Ln 27,Col 2: Syntax Error
	 Line no 27: '##'
ERROR   : Unable to parse `Class` markdown: '../model/Core/Classes/SpdxDocument.md'
WARNING : skipping ../model/Core/Properties/_property.md
WARNING : skipping ../model/Core/Vocabularies/_vocab.md
WARNING : Specs not parsed succesfully, aborting the gen_md...
WARNING : https://spdx.org/rdf/Build#map<string>string does not look like a valid URI, trying to serialize this will break.
WARNING : https://spdx.org/rdf/Build#map<string>string does not look like a valid URI, trying to serialize this will break.
WARNING : Error parsing the spec. Aborting the gen_rdf...
```

**Comment by zvr** [comment]
Is it not the same as #174 ? 

Copy-pasting: 

If it's running spec-parser, it would fail since the change to understand "External properties restrictions" has not been merged yet: spdx/spec-parser#39

**Comment by maxhbr** [comment]
duplicate of #174
